build.gradle

```diff
@@ -31,6 +32,10 @@ allprojects {
     }
 }
 
+tasks.withType(Javadoc).all {
+    enabled = false
+}
+
```

Comment:
Just move it to FishBun / build.gradle.

Reply:
This change will cause the build to fail.
i will search for another way.

Reply:
I have added code that currently does not generate Javadoc for Kotlin files in the develop branch. You can remove this code.

Reply:
Okay, I remove this code.